src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -1,18 +1,27 @@
-import { Box, color, Sans, Serif } from "@artsy/palette"
+import { color, Flex, Sans, Serif } from "@artsy/palette"
+import match from "autosuggest-highlight/match"
+import parse from "autosuggest-highlight/parse"
 import React, { SFC } from "react"
-
 interface Props {
   display: string
   label: string
+  query: string
 }
 
-export const SuggestionItem: SFC<Props> = ({ display, label }) => {
+export const SuggestionItem: SFC<Props> = ({ display, label, query }) => {
+  const matches = match(display, query)
+  const parts = parse(display, matches)
+
   return (
-    <Box p={1}>
-      <Serif size={"2"}>{display}</Serif>
-      <Sans color={color("black30")} size={"2"}>
+    <Flex pl={3} pb={3} justifyContent="center" flexDirection="column">
+      <Serif size="2">
+        {parts.map(({ highlight, text }, index) => {
+          return highlight ? <strong key={index}>{text}</strong> : text
+        })}
+      </Serif>
```

Comment:
This is cool, but I think we'll need something like this for other parts of the Onsite Search 2.0 project, so it's possible we'll want to generalize this somehow. We shouldn't do anything about this now, but something to keep in mind!!